## Welcome to my GitHub Pages

I am a final year PhD student under the Erasmus-Mundus Joint Doctorate Program in **Animal Breeding and Genetics** [EGS-ABG](http://www.egsabg.eu/), jointly hosted by **Aarhus University** [Center for Quantitative Genetics and Genomics (QGG)](http://mbg.au.dk/en/research/research-centres/center-for-quantitative-genetics-and-genomics/) and **AgroParisTech** [Génétique Animale et Biologie Intégrative (GABI)](https://www6.jouy.inra.fr/gabi/). 
The focus of my PhD project is the use of whole-genome sequences to identify recessive genetic defects in dairy cattle. 

Currently, I am working on following:
* deletion genotyping using Gaussian mixture model [GMM for genotypeing](http://pure.au.dk/portal/files/128029891/genotype_call_chromosomal_deletions_using_read_depth_whole_genome_sequence_variants_cattle.pdf) [read_Depth_genotyping](https://github.com/MMesbahU/ImputeDelPipeline/tree/master/read_Depth_genotyping)
* building haplotype reference for imputing chromosomal deletions [ImputeDelPipeline](https://github.com/MMesbahU/ImputeDelPipeline)
* GWAS and meta-analysis for female fertility using imputed whole-genome sequence variants and large deletions [GWAS](https://github.com/MMesbahU/gwas_in_cattle)
* Prediction of genomic breeding values 


## Research Interest
* Gene Mapping
* Genomic prediction using deep learning; Bayesian Modeling

## Publication
* **Mesbah-Uddin, M., et al.**, *A missense mutation (p.Tyr452Cys) in the CAD gene compromises reproductive success in French Normande cattle.* [J Dairy Sci, 2019](https://doi.org/10.3168/jds.2018-16100)
* **Mesbah-Uddin, M., et al.**, *Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle.* [DNA Res, 2018. 25(1): p. 49-59.](https://doi.org/10.1093/dnares/dsx037)
* **Mesbah-Uddin, M., et al.**, *In-silico analysis of inflammatory bowel disease (IBD) GWAS loci to novel connections.* [PLoS One, 2015. 10(3): p. e0119420.](https://doi.org/10.1371/journal.pone.0119420)
* **Mesbah-Uddin, M.**, *Prediction of deleterious nonsynonymous SNPs by integrating multiple classifiers – An application to neurodegenerative diseases.* [PeerJ PrePrints, 2015. 3: p. e994v1.](https://doi.org/10.7287/peerj.preprints.994v1)

### Conference
* **Mesbah-Uddin, M., et al.** *Genotype call for chromosomal deletions using read-depth from whole genome sequence variants in cattle. in Proceedings of the World Congress on Genetics Applied to Livestock Production.* 2018. Auckland, New Zealand.[**Paper**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/Mesbah-Uddin%2C%20M.%20et%20al.%20WCGALP%202018.pdf) [**Poster**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/poster_MMU%20et%20al%20WCGALP2018.pdf)
* **Mesbah-Uddin, M., et al.**, *A population scale discovery of large deletions in Holstein, Jersey and Nordic Red Cattle genome*, [in 36th International Society for Animal Genetics Conference. 2017: Dublin, Ireland.](http://www.isag.us/Docs/Proceedings/ISAG2017_Proceedings.pdf?v3) [**Poster**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/Mesbah-Uddin_et_al_2017_36th_ISAG_Dublin_Ireland.pdf)
