## Welcome to my GitHub Pages
In September 2019, I finished my [PhD in **Genetics**](http://pure.au.dk/portal/en/persons/md-mesbah-uddin(fa7659dc-95ae-430a-a82a-1aa9515ec6fd).html) under the Erasmus-Mundus Joint Doctorate Program [EGS-ABG](http://www.egsabg.eu/), jointly hosted by **Aarhus University** [Center for Quantitative Genetics and Genomics (QGG)](http://mbg.au.dk/en/research/research-centres/center-for-quantitative-genetics-and-genomics/) and **AgroParisTech** [Génétique Animale et Biologie Intégrative (GABI)](https://www6.jouy.inra.fr/gabi/). 
The focus of my PhD project was the use of whole-genome sequences to identify causal variants for recessive genetic defects in dairy cattle.
The title of the PhD project: ["Identification of Causal Factors for Recessive Lethals in Dairy Cattle with Special Focus on Large Chromosomal Deletions"](https://pure.au.dk/portal/en/persons/md-mesbah-uddin(fa7659dc-95ae-430a-a82a-1aa9515ec6fd)/publications/identification-of-causal-factors-for-recessive-lethals-in-dairy-cattle-with-special-focus-on-large-chromosomal-deletions(b8ea91a8-3983-4cb6-98f1-e599f80bb194).html)
### Ongoing Projects:
.......to be filled soon :-) ......
 
### Recently finished Projects
* [Genomic Prediction](https://github.com/MMesbahU/genomic_prediction.git) Prediction of genomic breeding values using imputed SNPs, indels and large chromosomal deletion
* [GWAS and meta-analysis](https://github.com/MMesbahU/gwas_in_cattle) for female fertility using imputed whole-genome sequence variants and large deletions
* Deletion genotyping using Gaussian mixture model [GMM](https://github.com/MMesbahU/ImputeDelPipeline/tree/master/read_Depth_genotyping)
* Building haplotype reference for imputing chromosomal deletions [ImputeDelPipeline](https://github.com/MMesbahU/ImputeDelPipeline)

## Research Interest
* I am interested in precision health care using genomics and machine learning appraoches: 
  * Gene Mapping
  * Genomic prediction: using Frequentist, Bayesian and Deep Learning approaches

## Publication
### Published/Accepted
* **Mesbah-Uddin, M.**, B. Guldbrandtsen, M. S. Lund, D. Boichard and G. Sahana. *Joint imputation of whole-genome sequence variants and large chromosomal deletions in cattle.* J Dairy Sci, 2019 (accepted)
* Wu, X., **M. Mesbah-Uddin**, B. Guldbrandtsen, M. S. Lund and G. Sahana. *Haplotypes responsible for early embryonic lethality detected in Nordic Holstein.* [J Dairy Sci, 2019](https://doi.org/10.3168/jds.2019-16651)
* Hoze, C., C. Escouflaire, **M. Mesbah-Uddin**, A. Barbat, M. Boussaha, M. C. Deloche, D. Boichard, S. Fritz, and A. Capitan. *Short Communication: A splice site mutation in CENPU is associated with recessive embryonic lethality in Holstein cattle.* J Dairy Sci, 2019 (accepted)
* **Mesbah-Uddin, M., et al.**, *A missense mutation (p.Tyr452Cys) in the CAD gene compromises reproductive success in French Normande cattle.* [J Dairy Sci, 2019](https://doi.org/10.3168/jds.2018-16100)
* **Mesbah-Uddin, M., et al.**, *Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle.* [DNA Res, 2018. 25(1): p. 49-59.](https://doi.org/10.1093/dnares/dsx037)
* **Mesbah-Uddin, M., et al.**, *In-silico analysis of inflammatory bowel disease (IBD) GWAS loci to novel connections.* [PLoS One, 2015. 10(3): p. e0119420.](https://doi.org/10.1371/journal.pone.0119420)
* **Mesbah-Uddin, M.**, *Prediction of deleterious nonsynonymous SNPs by integrating multiple classifiers – An application to neurodegenerative diseases.* [PeerJ PrePrints, 2015. 3: p. e994v1.](https://doi.org/10.7287/peerj.preprints.994v1)


### In Preparation:
* **Mesbah-Uddin, M.**, B. Guldbrandtsen, A. Capitan, M. S. Lund, D. Boichard and G. Sahana. *Genome-wide association study with imputed whole-genome sequence variants including large deletions for female fertility in three Nordic dairy breeds.*
* **Mesbah-Uddin, M.**, A. Capitan, B. Guldbrandtsen, M. S. Lund, G. Sahana and D. Boichard. *Genomic prediction for female fertility using imputed whole-genome sequence variants including large chromosomal deletions.*
### Conference
* **Mesbah-Uddin, M., et al.** *Genotype call for chromosomal deletions using read-depth from whole genome sequence variants in cattle. in Proceedings of the World Congress on Genetics Applied to Livestock Production.* 2018. Auckland, New Zealand.[**Paper**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/Mesbah-Uddin%2C%20M.%20et%20al.%20WCGALP%202018.pdf) [**Poster**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/poster_MMU%20et%20al%20WCGALP2018.pdf)
* **Mesbah-Uddin, M., et al.**, *A population scale discovery of large deletions in Holstein, Jersey and Nordic Red Cattle genome*, [in 36th International Society for Animal Genetics Conference. 2017: Dublin, Ireland.](http://www.isag.us/Docs/Proceedings/ISAG2017_Proceedings.pdf?v3) [**Poster**](https://github.com/MMesbahU/MMesbahU.github.io/blob/master/pdfs/Mesbah-Uddin_et_al_2017_36th_ISAG_Dublin_Ireland.pdf)

## Github repo:
* [ImputeDelPipeline](https://github.com/MMesbahU/ImputeDelPipeline): Pipelines for read-depth genotyping, building haplotype reference panel and imputing large chromosomal deletions along with WGS SNPs and indels
* [GWAS](https://github.com/MMesbahU/gwas_in_cattle): collections of scripts I'm using for GWAS and Meta-analysis
* [GenomicPrediction](https://github.com/MMesbahU/genomic_prediction.git)

